src/test/java/com/hazelcast/kubernetes/KubernetesClientTest.java

```diff
@@ -437,6 +441,26 @@ public void endpointsByNamespaceWithNodePublicIp() {
         assertThat(formatPublic(result), containsInAnyOrder(ready("35.232.226.200", 31916), ready("35.232.226.201", 31917)));
     }
 
+    @Test
+    public void endpointsByNamespaceWithNodeName() {
+        // given
+        // reuse setup from another test case
+        endpointsByNamespaceWithNodePublicIp();
```

Comment:
You call another test in this test, that is not right. Please extract the common parts to a method to reuse.

Reply:
True that. Extracted the JSON for the node port services into helper methods.